package/MDAnalysis/analysis/density.py

```diff
@@ -369,6 +369,7 @@ class DensityAnalysis(AnalysisBase):
     .. versionchanged:: 2.0.0
        :func:`_set_user_grid` is now a method of :class:`DensityAnalysis`.
     """
+
```

Comment:
I credit this change to - autopep8

Reply:
The choice to use an automatic formatter is kind of a personal choice (I don't because of issues like these, I prefer having fine grained control about how I will format things). If you do use one though, particularly in highly collaborative projects like MDAnalysis, it's on you to then check if the changes made were appropriate.

That being said, whilst my PEP8 linters don't identify issues with this, it's possible that there is some. Given that we have a lot of heterogeneity in whether or not we have a blank line between the docstring and `__init__`, is there something in the PEP8 documentation that indicates whether or not this is necessary?

Reply:
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings 
I found this which says it is recommended to add a line after all docstrings.

Reply:
That looks like it's for PEP 257 though

Reply:
That's fine. The importance here was to have a rationale to explain why a change has been made rather than relying on the accuracy of automated tools.

Reply:
Yes, probably not a good idea to make blind changes..